iphone/metadata/en-US/release_notes.txt

```diff
@@ -1,28 +1,19 @@
-• New OSM maps data as of February 4, 2022
-• New Help/About dialog on the main screen
-• Hide bottom toolbar on tap at the empty map
-• Transparent bottom toolbar background
-
-Routing:
-• Fixed routing bugs with unnecessary exits from the highway
-• Fixed some cyclist routing bugs
-• Respect "Avoid roads" setting for bicycle and pedestrian routes
-• Fixed bug with invalid cross-region routing
-
-Map:
-• Added search categories for Water, RV Facilities and Recycling
-• Added parcel pickup, excrement bags, quay, marina, caravan, dump station, mast, flag pole, funeral directors
-• Better displaying of house numbers in navigation
-• Select linear objects instead of area ones on long tap
-• Towns and cities are properly prioritized on 10-11 zoom levels
-• Prioritize parking over other areas like forests
-• Improved rendering of piers, dams, cutlines
-• Improved train stations visibility
-• Made national parks & military fills less obtrusive
-• Adjusted highways visibility
-• Fixed some capitals incorrectly marked as cities on a World map.
-• Corrected some icons
-
-Translations:
-• Fixed German, Italian and Turkish translations
-• FAQ in Turkish
+• New OSM data as of March 14, 2022
+• Display Wikipedia articles for well-known places
+• Fixed appearance of date pickers when editing opening hours
+• Fixed rare crashes after downloading maps
+• Select a proper light/dark CarPlay theme on the startup
+• Automatically switch CarPlay dark/light mode based on the system notification
+• Add date and time to My Position bookmarks
```

Comment:
it was done in previous release for both Android and iOS, right?
#1884

Reply:
No. It was merged after iOS release was published.